style.css

```diff
@@ -916,20 +906,35 @@ body {
   }
 
   #header {
-    top: 92px;
+    width: 100%;
+    height: 100%;
     display: none;
   }
 
   #appbar {
     display: block;
-    position: absolute;
+    position: fixed;
     width: 100%;
     height: 92px;
     background: #3c3a39;
+    z-index: 3;
   }
 
   #item_nav {
     display: none;
+    position: fixed;
+    left: 0;
+    right: 0;
+    top: -180px;
+    bottom: 0;
+    padding-left: 10px;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
```

Comment:
- [ ] As requested by the previous reviewer, **please align your nav item links more to the top, not in the middle.** Here is a quick comparison 👇 
Given design
![image](https://user-images.githubusercontent.com/90222110/208712256-508b872a-ee8c-488e-a44f-683b74cb75bb.png)

Your design
![image](https://user-images.githubusercontent.com/90222110/208712402-ff03a3cb-e291-4c33-8924-60472c7b8530.png)

HINT:
I would suggest adjusting the following properties but It's up to you to align them correctly and make sure nothing else is breaking 😃 👍 
- justify-content: flex-start;
- top: 89px

Reply:
Its already aligned on my screen